src/scripts/oakItems/OakItems.ts

```diff
@@ -38,6 +38,15 @@ class OakItems implements Feature {
         return oakItem.calculateBonus();
     }
 
+    calculateBonusIfActive(item: OakItems.OakItem) {
```

Comment:
This is the same as calculateBonus, but just calls something different at the end. Feels like we could just be passing some `pretendActive` parameter to calculateBonus instead.

Reply:
On further thought, I think also passing some number `bonusModifier` would be good, to have the oak item do this extra math for us. (with the intention to be used as a 0-1 value to scale the amount of bonus we actually get)